README.md

```diff
@@ -0,0 +1,43 @@
+# Babelfish Tools
+
+[![Build Status](https://travis-ci.org/bblfsh/tools.svg?branch=master)](https://travis-ci.org/bblfsh/tools)
+[![codecov](https://codecov.io/gh/bblfsh/tools/branch/master/graph/badge.svg)](https://codecov.io/gh/bblfsh/tools)
+
+Language analysis tools on top of Babelfish
+
+## Build
+
+### With docker
+
+`make build`
+
+### Without docker
+
+`make build-internal`
```

Comment:
I'm not sure this is the best way to build it without if you don't want to use docker. Should `go build` be enough?